assignments/003-illumina-sequencing-questions/illumina-seq-homework.md

```diff
@@ -0,0 +1,56 @@
+
+
+These are the questions that appear in the video about Illumina sequencing by synthesis that
+can be viewed at: [https://eriqande.github.io/erics-captioned-vids/vids/illumina-sbs/](https://eriqande.github.io/erics-captioned-vids/vids/illumina-sbs/).
+
+Copy this template to a new file in the same directory called simply
+`illumina-seq-homework.md`.  Please answer each question by typing in the space below each question, or by retaining the words corresponding to your answers in the "dual-choice in parentheses" questions.
+
+When you are done.  Save the file then commit it
+to a branch called `illumina-seq` and push it back to your
+fork of the course repo on GitHub then send me a pull request.
+
+
+
+* Q1: True or False. This animation is showing that each of the two "fragments" is a double-stranded segment of DNA. (_from 30.0 to 33.0 seconds in the video_).
+False, each fragment is a single strand of DNA
+
+
+* Q2: How does the sequence of the fragment on top compare to the sequence of the fragment on the bottom? (_from 36.5 to 39.0 seconds in the video_).
+They are complementary.
+
+
+* Q3: What is the purpose of the sequencing binding sites? Your answer should include the words "oligonucleotide," "polymerase," "3'-hydroxyl," and "synthesize." (_from 39.0 to 40.5 seconds in the video_).
+
+
+
+* Q4: Given the direction of travel of the polymerase, is the Original Template anchored to the lawn by its 3' or its 5' end? (_from 78.0 to 81.0 seconds in the video_).
+It would be anchored by it's 3' end.
+
+
+* Q5: How does the remaining strand relate to the Original Template? Is the remaining strand anchored by its 3' or 5' end? (_from 85.0 to 88.0 seconds in the video_).
+The remaining strand is complementary, and it's anchored by it's 5' end.
+
+
+* Q6: Choose one from the parentheses: the purple part on the remaining strand is (the same as || complementary to) the purple lawn oligos.  (_from 88.2 to 92.2 seconds in the video_).
+complementary to
```

Comment:
It's actually the same as.  Lots of people missed this---I probably asked the question in a confusing way